ui/tract-renderer/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="previous">previous</string>
```

Comment:
let's name these something more unique. when phrases are translated to other languages they can be different based on their context. In order to allow translators to do this we need to make sure we allow multiple "next" phrases based on where we use them in the app. the easiest way is to use some context in phrase names. so the phrase should probably be `tract_card_next` and `tract_card_previous`